webpack.config.js

```diff
@@ -0,0 +1,24 @@
+var path = require('path')
+var webpack = require('webpack')
+
+module.exports = {
+  entry: "./static/components/PoiCards.js",
+  output: {
+    path: "./static/components",
+    filename: "App-around.js"
+  },
+  watch: true,
+
+  module: {
+    loaders: [
+      {
+        test: /.jsx?$/,
```

Comment:
If it was necessary but a mystery, you should have added the link that was explaining that requirement so that the next developer would understand why you did it.

No applicable here as it seems like unrequired.